clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ClioIndex.scala

```diff
@@ -51,6 +51,8 @@ trait ClioIndex {
 
   val queryInputMapper: FieldMapper[QueryInputType]
 
+  val defaults: Json
```

Comment:
I'm not convinced this should go into `ClioIndex`. All this code deals with client <-> server API, but this is a detail of recovery buried in the backend. (Though we do have `elasticsearchIndexName` in this trait, but I continue to object to that too).